web/src/app/main/themeConfig.tsx

```diff
@@ -41,24 +40,37 @@ function handleStoreThemeMode(theme: string): boolean {
   return false
 }
 
+// palette generated from https://material-foundation.github.io/material-theme-builder/#/custom
 function getPalette(mode: string): PaletteOptions {
   const prefersDark = window.matchMedia('(prefers-color-scheme: dark)')
 
-  if (mode === 'dark' || (mode === 'system' && prefersDark)) {
+  if (mode === 'light' || (mode === 'system' && !prefersDark)) {
     return {
-      mode: 'dark',
-      secondary: grey,
+      mode: 'light',
+      primary: {
+        main: '#006684',
+        light: '#bbe9ff',
+        dark: '#001f2a',
+      },
+      secondary: { main: '#4d616b', light: '#d0e6f3', dark: '#081e27' },
+      background: {
+        default: '#fbfcfe',
+        paper: '#dce3e8', // m3 surface variant
+      },
+      error: { main: '#ba1b1b', light: '#ffdad4', dark: '#410001' },
     }
   }
 
-  if (mode === 'light' || (mode === 'system' && !prefersDark)) {
+  if (mode === 'dark' || (mode === 'system' && prefersDark)) {
     return {
-      mode: 'light',
-      primary: {
-        ...grey,
-        main: '#616161',
+      mode: 'dark',
+      primary: { main: '#64d3ff', light: '#bbe9ff', dark: '#004d65' },
+      secondary: { main: '#b5cad6', light: '#d0e6f3', dark: '#354a54' },
+      background: {
+        default: '#191c1e',
+        paper: '#191c1e', // m3 surface
       },
-      secondary: grey,
+      error: { main: '#ffb4a9', light: '#ffdad4', dark: '#930006' },
```

Comment:
Hmm, looks like it's grabbing the wrong value here, so I'll look into that. I assumed the text for dark mode would show as #FFB4A9, the main error color

<img width="151" alt="Screen Shot 2022-03-08 at 2 15 07 PM" src="https://user-images.githubusercontent.com/11381794/157335041-fba3379b-0d74-4b7a-ad17-087adb4f8bbe.png">

<img width="317" alt="Screen Shot 2022-03-08 at 2 14 19 PM" src="https://user-images.githubusercontent.com/11381794/157334993-fbda193e-9833-4919-a0f8-f7f07f626cb4.png">

<img width="582" alt="Screen Shot 2022-03-08 at 2 14 49 PM" src="https://user-images.githubusercontent.com/11381794/157335007-762d945e-e59f-4c5b-bc1f-f8f345bb5360.png">

Reply:
Looks like this is coming from a hardcoded value that you are addressing in a separate PR, so would probably be out of scope for this bit since it's an issue in master as well @dctalbot

Reply:
This was resolved by #2218.

<img width="380" alt="Screen Shot 2022-03-10 at 8 23 12 AM" src="https://user-images.githubusercontent.com/11381794/157708366-d185de73-4af8-4b2b-8d13-f8faa429e22c.png">